src/site/topics/recursion/recursion.rst

```diff
@@ -32,62 +29,87 @@ Groups of People
 * Like the line of people example above, it may seem like a silly question because it is so intuitive
 
 * A group of people is:
+
     * 2 people
     * OR 3 people
     * OR 4 people
     * OR ...
     * ...
     * ...
 
-* Alternatively, we can use the idea of recursion to define a group of people a different way
-* When we use recursion to define something, we use the definition of itself in it's definition
+* Alternatively, the idea of recursion can be used to define a group of people a different way
+* When defining something recursively, the definition of something is used within its definition
+
     * Define something in terms of itself
 
-* Recursive definition of a **group of people** is:
+
+* A recursive definition of a **group of people** is:
+
     * 2 people
     * OR a **group of people** plus one more person
```

Comment:
Maybe worth relating it to an inductive proof for the math nerds out there. :shrug:

Reply:
I like the idea of directly tying it to induction, but I fear this particular example is too trivial that it will come across annoying more than anything else

![image](https://user-images.githubusercontent.com/1328048/217598831-c857a7a7-fe2a-4969-a9ee-b2dff143030f.png)

Reply:
I'm going to talk to my math prof buddy to see if we can come up with an interesting example